lib/src/chart/line_chart/line_chart_renderer.dart

```diff
@@ -32,12 +39,19 @@ class LineChartLeaf extends LeafRenderObjectWidget {
 
 /// Renders our LineChart, also handles hitTest.
 class RenderLineChart extends RenderBaseChart<LineTouchResponse> {
-  RenderLineChart(BuildContext context, LineChartData data,
-      LineChartData targetData, double textScale)
-      : _data = data,
+  RenderLineChart(
```

Comment:
These changes can be reverted. Please revert them to keep the PR as clean as possible.